kokoro/xds.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -exu -o pipefail
+if [[ -f /VERSION ]]; then
+  cat /VERSION
+fi
+
+cd github
+
+export GOPATH=~/gopath
+REPO_ROOT=${GOPATH}/src/google.golang.org/grpc
+
+mkdir -p ${GOPATH}/src/google.golang.org
```

Comment:
Do you know which go version this script will run with?

If the go version supports modules (>= go1.11), you don't need to move the source to this specific directory in gopath. You can just cd to grpc and run build.

```sh
export GOPATH=~/gopath
# assuming source is cloned to directory grpc-go
pushd grpc-go/interop/xds/client
go build
popd
```

Reply:
Done, we have go >= 1.11